src/api/exchange/ExchangeApi.ts

```diff
@@ -105,7 +105,16 @@ export interface ExchangeApi extends DepositApi {
   cancelOrders(params: CancelOrdersParams): Promise<Receipt>
 }
 
-export interface DetailedAuctionElement extends AuctionElement {
+// Where market = 'WETH-TUSD'
+export interface Market {
```

Comment:
Do we need to introduce artificial derived data just for the filter?
Can't we just define this logic in the filter function itself?

This way, also the filtering by market can have self contained in the function the logics like: 
- accept: WETH-TUSD
- accept: WETHTUSD
- accept: WETH/TUSD

Reply:
@anxolin fixed in #1119 via commits [129c9c](https://github.com/gnosis/dex-react/pull/1119/commits/129c9c12b2da48410c2d4c4d3bc79b9b294f603a) and [
ab85b52](https://github.com/gnosis/dex-react/pull/1119/commits/ab85b52a7acaee7c53007f16350668567d657d58)